addon/mixins/osf-cookie-login-route.js

```diff
@@ -15,13 +13,17 @@ import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-
  * @extends Ember.Mixin
  * @uses ember-simple-auth/UnauthenticatedRouteMixin
  */
-export default Ember.Mixin.create(UnauthenticatedRouteMixin, {
```

Comment:
In the current cookie use case, there is no such thing as a "cookie login route" at all- it's simply mixed into the application route. (Your cookie login route is actually a CAS or flask app URL, not in ember at all)

This allows us to do things like checking for a valid cookie on first page load (so navbar shows correctly logged in state immediately).

From http://ember-simple-auth.com/api/classes/UnauthenticatedRouteMixin.html
> http://ember-simple-auth.com/api/classes/UnauthenticatedRouteMixin.html